b-real-time/b-real-time-htb.js

```diff
@@ -72,11 +59,25 @@ function BRealTimeHtb(configs) {
      *
      * @private {string}
      */
-    var __endpoint = '//ib.adnxs.com/ut/v2/prebid';
+    var __endpoint = '//hb.emxdgt.com/';
 
     /* =====================================
      * Functions
      * ---------------------------------- */
+    function __getMatchingBid(parcel, bidResponse) {
+        if (!bidResponse.hasOwnProperty('seatbid')) {
+            return false;
+        }
+        for (var emxBid of bidResponse.seatbid) {
```

Comment:
Please update this loop from a `for .. of` loop to a normal `for` loop as this does not correctly parse in IE.

Reply:
Done